src/fromager/resolver.py

```diff
@@ -36,7 +37,7 @@
 
 def get_project_from_pypi(
     project: str,
-    extras: tuple[str],
+    extras: tuple[str] | set[str],
```

Comment:
`tuple[str]` is a tuple with one item. Candidate expects an `Iterable[str]`.

```suggestion
    extras: typing.Iterable[str],
```